Religion Functionality, also Molten Core

Wymrite opened the pull request:
Added automation options for building all Ziggurat and Cryptotheology buildings in the Religion tab along with Molten Core in the Space tab. Currently the unicorn building automation isn't super useful as it just buys the cheapest as usual, though this might be nice for longer idling sessions.

cameroncondry commented on kitten-scientists.user.js:
```diff
@@ -209,7 +230,8 @@ var run = function() {
                     // Centaurus
-                    tectonic: {require: 'antimatter', enabled: false}
+                    tectonic:      {require: 'antimatter', enabled: false},
+                    moltenCore:    {require: 'uranium',     enabled: false}
```
Please keep spacing consistent with the rest of the file to keep code looking clean.

cameroncondry commented on kitten-scientists.user.js:
```diff
@@ -712,25 +734,39 @@ var run = function() {
+				activity('Kittens have discovered ' + build.label, 'ks-faith');
+			} else {
+				activity('Kittens have built a new ' + build.label, 'ks-faith');
+			}
```
It looks like there is extra tabs being added before the code. Align the code with the rest of the function to keep code easy to read.

cameroncondry commented on kitten-scientists.user.js:
```diff
@@ -712,25 +734,39 @@ var run = function() {
+		getBuild: function (name, variant) {
+			if (variant === "z"){
+				return game.religion.getZU(name);
+			}else if (variant === "s"){
```
I know this isn't completely consistent across the file, however, I prefer we use the following layout for if/else statements:

```javascript
if (condition) {
    // results true
} else {
    // results false
}
```

If there are more than 4 cases, then consider using a switch case instead. In this case I will leave it up to your discretion since there are only 3 cases.

Wymrite commented:
Formatting should be all fixed now, turns out I had misunderstood how the GitHub tab button options function. I'm not certain on the formatting for the actual build options list, since it seems to differ somewhat between sections, but the religion build options are at least far more readable now. Changed those two if/else statements to switch statements since someone may want to separate out the unicorn production specific buildings in the future. Also added a comment to explain the variants at the build options section.

Wymrite commented:
@cameroncondry Made some final formatting fixes just now, should now be fully consistent with the rest of the code. Been running this version for about 2 days now while experimenting with some time tab stuff and all the new religion autobuilding appears to be fully functional.

cameroncondry commented:
Most excellent! Those changes look perfect. After reading your comments, the variant property makes perfect sense. Merging the changes in now. Thanks again!
